src/kbmod/search/KBMOSearch.cpp

```diff
@@ -314,6 +314,55 @@ std::vector<RawImage> KBMOSearch::meanScienceStamps(const std::vector<Trajectory
     return (results);
 }
 
+std::vector<RawImage> KBMOSearch::coaddedScienceStampsGPU(std::vector<trajectory>& t_array,
+                                                          int radius, bool compute_mean) {
+    const int num_images = stack.imgCount();
+    const int width = stack.getWidth();
+    const int height = stack.getHeight();
+
+    // Copy the image pixels into a single array.
+    unsigned int ppi = width * height;
```

Comment:
how often does this get called compared to how often the result is used? Might be a couple of extra copies involved here (or not, hard to tell from just this code).

Reply:
It won't be called too often, but I would like to remove any unneeded copies.

The specific line you are commenting on seems to be from an old copy of the PR though. I removed the copy of image pixels into a single array part.